src/cls/rgw/cls_rgw_ops.h

```diff
@@ -554,23 +554,16 @@ WRITE_CLASS_ENCODER(rgw_cls_obj_check_mtime)
 
 struct rgw_cls_usage_log_add_op {
   rgw_usage_log_info info;
-  rgw_user user;
 
   void encode(bufferlist& bl) const {
-    ENCODE_START(2, 1, bl);
+    ENCODE_START(3, 1, bl);
     ::encode(info, bl);
-    ::encode(user.to_str(), bl);
     ENCODE_FINISH(bl);
   }
 
   void decode(bufferlist::iterator& bl) {
-    DECODE_START(2, bl);
+    DECODE_START(3, bl);
     ::decode(info, bl);
-    if (struct_v >= 2) {
-      string s;
-      ::decode(s, bl);
-      user.from_str(s);
-    }
```

Comment:
@yehudasa I've removed this block after your suggestion for  `ENCODE_START(3, 3, bl)` . As we've changed [compat](https://github.com/ceph/ceph/blob/master/src/include/encoding.h#L1025) to 3,  this block would not be executed?